x-pack/solutions/security/plugins/security_solution/common/siem_migrations/model/rule_migration.gen.ts

```diff
@@ -71,6 +71,10 @@ export const OriginalRule = z.object({
    * The original rule annotations containing additional information.
    */
   annotations: OriginalRuleAnnotations.optional(),
+  /**
+   * The original rule severity.
+   */
+  severity: z.string().optional(),
```

Comment:
I miss the changes to the 
`x-pack/solutions/security/plugins/security_solution/common/siem_migrations/model/rule_migration.schema.yaml`

We should not edit the `*.gen.ts` files manually. Let's edit the yaml and execute:
```
node scripts/generate_openapi --rootDir x-pack/solutions/security/plugins/security_solution/common/siem_migrations/
```

And, it would be better if we set the correct type here: 
```yml
        severity:
          type: string
          description: The original rule severity.
          enum: 
            - '1'
            - '2'
            - '3'
            - '4'
            - '5'
```
So we don't have to cast it with `as SplunkSeverity` later.

Reply:
Now that I am thinking, 🤔 better let's not assume a specific enum for all vendors, let's keep a generic string and cast it later 👍

Reply:
💯 . that is what I was replying. Let me add changes to yaml

Reply:
done here: https://github.com/elastic/kibana/pull/211202/commits/979645ea97e1ad12313431d07607764d268be9f4